spec/spec_helper.rb

```diff
@@ -2,6 +2,7 @@
 require "rspec"
 require "rack/test"
 require "rate_limiting"
+require "pry"
```

Comment:
do we still require pry here?

Reply:
I think so. I couldn't use it without the require. Rails does a lot of magic includes that plain old apps don't.